### DISCUSSION TITLE
Console: Run SQL HTML results are now div instead of table; can't copy/paste into spreadsheet editor anymore

### DISCUSSION THREAD

**Discussion by carlpaten** [hasura/graphql-engine#8855] [Question]
### Version Information

Server Version: v2.4.0
CLI Version (for CLI related issue): v2.4.0

### Environment

OSS

### What is the current behaviour?

When using the Run SQL page, tabular results are returned as a series of nested divs, which cannot be pasted into a spreadsheet editor without losing the table structure

### What is the expected behaviour?

As in Hasura 1.x, the Run SQL page should show tabular results using HTML `<table>`

### How to reproduce the issue?

1. Visit `/console/data/sql`
2. `SELECT` some multi-row, multi-column dataset
3. Inspect results; view that the table is made of `<div>`s
4. Try to paste into Google Sheets and see that it doesn't work

### Keywords

hasura web console run sql html table div copy paste google sheets microsoft excel

**Comment by ajohnson1200** [comment]
@carlpaten super easy workaround: click Export Data --> Export Data CSV and then pipe that into Google Sheets?

**Reply by carlpaten** [reply]
Export Data CSV seems to be broken, I can never get more than a small number of records. Also, copying from results used to allow a quick and easy way to extract ad-hoc data sets; Export Data CSV, even if it worked, would require the extra steps of creating a view and then dropping it afterwards. I'm not a big fan!

**Comment by robertjdominguez** [comment]
Converting this to a discussion as you all talk through @ajohnson1200's suggestion 👍

**Reply by carlpaten** [reply]
@robertjdominguez this is definitely a bug report/feature request (the line can be blurry sometimes) rather than a request for assistance.

**Reply by dsandip** [reply]
@carlpaten 💯 agreed. Will move it back to an issue and also follow up on the Export CSV workflow being broken. cc: @meetzaveri

**Reply by dsandip** [reply]
@carlpaten Also, sorry about ping-pong-ing this :)

**Reply by carlpaten** [reply]
No worries, thank you so much for your attention.

**Reply by meetzaveri** [reply]
As suggested by @dsandip , I am creating issue from this discussion and will lock this discussion
